integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -698,6 +702,35 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
     assertResult("The table has 'preaggregate' DataMap, it doesn't support streaming")(msg.getMessage)
   }
 
+  test ("handoff segment to update mergedto segment value") {
```

Comment:
suggest merging this test case into test case "auto hand off, close and reopen streaming table"
because streaming ingestion will take more time.